feature-libs/product/configurators/rulebased/cpq/cpq.models.ts

```diff
@@ -83,6 +79,18 @@ export namespace Cpq {
     quantity?: string;
   }
 
+  /**
+   *
+   * An interface representing the structure for update of CPQ configuration attribute.
+   */
+  export interface UpdateAttribute {
+    configurationId: string;
+    stdandardAttrCode: string;
```

Comment:
typo: do you mean **standardAttrCode**? Why not simply **attrCode**?

Reply:
This is the name of the CPQ API parameter

Reply:
UpdateAttribute structure is adjusted

Reply:
@Larisa-Staroverova: @AlexanderKuschner. I checked swagger:
https://eudev.webcomcpq.com/webapihelp/index#!/Configurations/Configurations_ChangeAttributeValue
st**d**andardAttrCode is indeed a typo! correct value is standardAttrCode